app/models/foreman_fog_proxmox/proxmox_vm_commands.rb

```diff
@@ -39,25 +39,32 @@ def create_vm(args = {})
       raise ::Foreman::Exception, format(N_('invalid vmid=%<vmid>s'), vmid: vmid) unless node.servers.id_valid?(vmid)
 
       image_id = args[:image_id]
-      name = args[:name]
       remove_volume_keys(args)
-      parsed_args = parse_typed_vm(args, type)
+      poolid = args[:pool] if args.key?(:pool)
```

Comment:
the poolid is only necessary if image_id is necessary? if this is the case, should we put this in the image_id block?

Reply:
added the change